src/components/modal_shared/Modal.sc.js

```diff
@@ -40,6 +110,23 @@ const Icon = styled.img`
     `}
 `;
 
+const Footer = styled.footer`
```

Comment:
a discussion that we should have once: should this be a little Footer.sc.js near Footer.js? And in general, should we always have the styled-component definition near a reusable component's js definition? My intuition is that it would make component editing easier. You alwyas know where to look. But we'll have more files.